payment/src/main/java/com/kyriba/conference/payment/domain/vo/Amount.java

```diff
@@ -0,0 +1,19 @@
+package com.kyriba.conference.payment.domain.vo;
```

Comment:
Value Object is also DTO, so the same package could be used.

Reply:
done